crates/swc/tests/tsc-references/classDeclarationLoop.2.minified.js

```diff
@@ -1,9 +1,8 @@
 //// [classDeclarationLoop.ts]
 import _class_call_check from "@swc/helpers/src/_class_call_check.mjs";
 for(var _loop = function(i) {
-    var C = function C() {
+    arr.push(function C() {
         "use strict";
         _class_call_check(this, C), this.prop = i;
-    };
-    arr.push(C);
+    });
```

Comment:
I see. In this case `arr` used to be marked as `var_kind` `None`.

Reply:
Yeah and that's why it was not inlined I guess